src/runtime/linux.rs

```diff
@@ -796,31 +798,39 @@ pub struct LinuxRdma {
     hca_objects: Option<u32>,
 }
 
-#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, Hash)]
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, Hash, StrumDisplay)]
 #[serde(rename_all = "snake_case")]
 /// Available Linux namespaces.
 pub enum LinuxNamespaceType {
+    #[strum(to_string = "mnt")]
     /// Mount Namespace for isolating mount points
     Mount = 0x00020000,
 
+    #[strum(to_string = "cgroup")]
     /// Cgroup Namespace for isolating cgroup hierarchies
     Cgroup = 0x02000000,
```

Comment:
Do we need to set this explicitly? I thought we just have to adapt the default value for `Mount` and `Network` and default to lower case transformation.

Reply:
What's about this setting about them ?
```diff
#[strum(serialize_all = "lowercase")]
#[serde(rename_all = "snake_case")]
/// Available Linux namespaces.
pub enum LinuxNamespaceType {
    #[strum(to_string = "mnt")]
    /// Mount Namespace for isolating mount points
    Mount = 0x00020000,

    /// Cgroup Namespace for isolating cgroup hierarchies
    Cgroup = 0x02000000,

    /// Uts Namespace for isolating hostname and NIS domain name
    Uts = 0x04000000,

    /// Ipc Namespace for isolating System V, IPC, POSIX message queues
    Ipc = 0x08000000,

    /// User Namespace for isolating user and group  ids
    User = 0x10000000,

    /// PID Namespace for isolating process ids
    Pid = 0x20000000,

    #[strum(to_string = "net")]
    /// Network Namespace for isolating network devices, ports, stacks etc.
    Network = 0x40000000,

    /// Time Namespace for isolating the clocks
    Time = 0x00000080,
}
```

Reply:
yeah, I have added related test cases for "net" and "mount/mnt". Please see the `ns_type_enum_to_string()` and `ns_type_string_to_enum() `